vue.js

```diff
@@ -0,0 +1,133 @@
+        '@typescript-eslint/consistent-type-definitions': [
+          'error',
+          'interface',
+        ],
+        '@typescript-eslint/explicit-function-return-type': 'error',
+        '@typescript-eslint/explicit-member-accessibility': [
+          'error',
+          {
+            accessibility: 'no-public',
+          },
+        ],
+        '@typescript-eslint/member-ordering': [
+          'warn',
+          {
+            classes: ['field', 'constructor', 'method'],
+          },
+        ],
+        '@typescript-eslint/naming-convention': [
+          'error',
+          {
+            selector: ['class', 'interface', 'typeAlias'],
+            format: ['PascalCase'],
+          },
+        ],
+
+        // For instance, when defining a void event emitter method in a component, then we don't want to get a linter warning.
+        '@typescript-eslint/no-empty-function': 'off',
+
+        'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+        'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+
+        'accessor-pairs': 'error',
+        'array-callback-return': 'error',
+        'capitalized-comments': ['warn', 'always'],
+        curly: 'error',
+        'default-case-last': 'error',
+        'default-param-last': 'error',
+        'dot-notation': 'error',
+        eqeqeq: ['error', 'always', { null: 'never' }],
+        'guard-for-in': 'error',
+        'no-constructor-return': 'error',
+        'no-else-return': 'error',
+        'no-extra-bind': 'error',
+        'no-lone-blocks': 'error',
+        'no-new-wrappers': 'error',
+        'no-nested-ternary': 'error',
+        'no-restricted-globals': [
+          'error',
+          { name: 'parseInt', message: `Use 'Number.parseInt()' instead.` },
+          { name: 'parseFloat', message: `Use 'Number.parseFloat()' instead.` },
+        ],
+        'no-self-compare': 'error',
+        'no-throw-literal': 'error',
+        'no-useless-rename': 'error',
+        'no-useless-return': 'error',
+        'import/no-cycle': 'error',
+        'import/order': [
+          'warn',
+          {
+            'newlines-between': 'always',
+            alphabetize: {
+              order: 'asc',
+              caseInsensitive: true,
+            },
+          },
+        ],
+        'no-unreachable-loop': 'error',
+        radix: 'error',
+        'require-atomic-updates': 'error',
+        'sort-imports': [
+          'warn',
+          {
+            ignoreDeclarationSort: true,
+          },
+        ],
+        'spaced-comment': ['warn', 'always'],
+        'valid-typeof': [
+          'error',
+          {
+            requireStringLiterals: true,
+          },
+        ],
+
+        'vue/html-self-closing': [
+          'warn',
+          {
+            html: {
+              void: 'always',
+              normal: 'never',
+              component: 'always',
+            },
+          },
+        ],
+      },
+    },
+  ],
+};
```

Comment:
Not sure, but in my case the issue was, that I do not have prettier installed globally and forgot to execute npm install -> prettier failed because it was not found